configure.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
Aren't these imports unneccessary given the syntax is valid in both 2 and 3?

Reply:
This import makes old style __print__ statements a syntax error in both Python 2 and Python 3.  This helps to make sure that Python 2 developers do not add code that will break Python 3.